packages/@lightningjs/ui-components/src/components/ProgressBar/ProgressBar.js

```diff
@@ -101,7 +100,7 @@ export default class ProgressBar extends Base {
   }
 
   get announce() {
-    if (this._announce) {
+    if (this._announce != undefined) {
```

Comment:
Updated the conditional form evaluating truthy/falsey of `this._announce` to explicitly check whether or not it is `undefined` or `null`. This allows setting `ProgressBar.announce` to `''` and still getting that empty string instead of its value when the announce getter is called.

The same change is made in `Slider`.